### PR TITLE
First version of the How To Make a Good README

### PR DESCRIPTION
**Pull request by esgg**

### CODE REVIEW

File: pages/your_tasks/how_to_make_a_good_readme.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    url:
+# More information on how to fill in this metadata section can be found here https://rdmkit.elixir-europe.org/page_metadata
+---
+<!-- Please take in mind our style guide https://rdmkit.elixir-europe.org/style_guide when writing the content of this page. -->
+
+## Why is it important a good README file? <!-- example: how to version control my code? -->
+ 
+### Description <!-- do not delete this heading and write your text below it -->
+
+The README file is the entry point to the documentation associated with a software. The main purpose of the README is:
```
Comment (anenadic) [inline pages/your_tasks/how_to_make_a_good_readme.md:21]:
```suggestion
The README file is the entry point to the documentation associated with a software. The main purpose of the README is to:
```

File: pages/your_tasks/how_to_make_a_good_readme.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+* Describe the structure of the project and its purpose.
+* Facilitate the installation and execution of the software stored in the repository.
+* Detail the use of the software.
```
Comment (anenadic) [inline pages/your_tasks/how_to_make_a_good_readme.md:24]: Point to the software's licence for the terms of reuse?

File: pages/your_tasks/how_to_make_a_good_readme.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+ 
+
+## Which sections should be addressed in a good README? <!-- example: how to name a software release? -->
+ 
+### Description <!-- do not delete this heading and write your text below it -->
+A README file is the first document that a researchers read when they are interested on using a software. For this reason, should contain all the relevant information to install and to execute it. It includes references to additional information such as API documentation, specifications, scientific publications related, tutorials, guidelines, etc.
+
+A basic software documentation emphasizes these aspects[^2] :
+* Understandability: a user can comprehend the structure and the purpose of the software.
+* Usability: a user can understant how to use the software.
```
Comment (anenadic) [inline pages/your_tasks/how_to_make_a_good_readme.md:45]:
```suggestion
* Usability: a user can understand how to use the software.
```

File: pages/your_tasks/how_to_make_a_good_readme.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+* Attribution: how a user must cite the software.
```
Comment (anenadic) [inline pages/your_tasks/how_to_make_a_good_readme.md:46]:
```suggestion
* Attribution: how a user should cite the software.
```

File: pages/your_tasks/how_to_make_a_good_readme.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+
+These aspects should be addressed in different sections of the README. 
+
+### Considerations <!-- do not delete this heading and write your text below it -->
+* A good README should describe the purpose and the features of the software.
+* A good README should describe how to execute and to use the code. 
+* In a research context, a good README should indicate how the software must be cited.
```
Comment (anenadic) [inline pages/your_tasks/how_to_make_a_good_readme.md:53]:
```suggestion
* In a research context, a good README should indicate how the software should be cited.
```

File: pages/your_tasks/how_to_make_a_good_readme.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+* In a research context, a good README should indicate the funding agency and the proper acknowledgements.
+* In the context of scientific experiments, a good README should facilitate its reproducibility.
```
Comment (anenadic) [inline pages/your_tasks/how_to_make_a_good_readme.md:55]:
```suggestion
* In the context of a scientific experiment, a good README should facilitate its reproducibility.
```

File: pages/your_tasks/how_to_make_a_good_readme.md [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+**Description**
+
+In this section, you will have to describe the purpose and the features of your software.
+
+**Requirements**
+
+In this section, you can include the requirements needed to install and to execute the software. These requirements can be software (e.g. operative system) or hardware (e.g. CPU, memory, etc.). There is no need to include low level requirements like language library dependencies. These requirements must be provided with the appropriate mechanism of each language. For example, in Python by using the requirements.txt file. 
+
+This is a non exhaustive list of requirements categories that you can consider:
+Operative System version
```
Comment (anenadic) [inline pages/your_tasks/how_to_make_a_good_readme.md:70]:
```suggestion
Operating system version
```

File: pages/your_tasks/how_to_make_a_good_readme.md [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+Operative System libraries version
```
Comment (anenadic) [inline pages/your_tasks/how_to_make_a_good_readme.md:71]:
```suggestion
Operating system libraries version
```

Comment (anenadic) [reply pages/your_tasks/how_to_make_a_good_readme.md:71]: Not sure what is meant by "Operating system libraries version" - is it the dependencies of the software?

File: pages/your_tasks/how_to_make_a_good_readme.md [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+
+If you have multiple options for installing the software (e.g. installing a library, docker, script) you should describe them.
+
+**Configuration**
+
+Sometimes, software needs to be configured to be executed. Usually, this task is done by filling in a configuration file. You will need to add this information to the README. In case you have documentation that describes the fields of the configuration file, you can also add it to the README.
+
+**Usage**
+
+In this section, you will include all the steps needed to use the application. For example, if it is a command line tool, you will need to describe all the parameters. As we mentioned in the installation section, you have to assume no skills in the user, so you will need to be exhaustive.
```
Comment (anenadic) [inline pages/your_tasks/how_to_make_a_good_readme.md:90]:
```suggestion
In this section, you should include all the steps needed to use the application. For example, if it is a command line tool, you will need to describe all the parameters. As we mentioned in the installation section, you have to assume no skills in the user, so you will need to be exhaustive.
```

File: pages/your_tasks/how_to_make_a_good_readme.md [hunk 10]
```diff
@@ -0,0 +1,125 @@
...
+
+Research software can be developed to execute experiments whose results are shown in a scientific publication. For this reason, and to add transparency to the research, it is important to show how the results can be achieved. You can use this section to describe the process to get the results. 
+
+**Contribution**
+
+Research software should be a collaborative effort, so a mechanism to allow other researchers to collaborate with the project is needed. In this section, you can describe how this mechanism is. For example, if you want to add a new feature, you can create a pull request to integrate this new feature.
+
+**Acknowledgement**
+
+Funders of the research software must be mentioned in the README. There will be a high variety of funders, from public/private institutions to national/international governments. You will have to include all of them. Check the acknowledgement policy of each institution to do it properly.
```
Comment (anenadic) [inline pages/your_tasks/how_to_make_a_good_readme.md:102]:
```suggestion
All funders of the research software should be mentioned in the README - check the acknowledgement policy of each institution for how best to do it.
```

**Review by anenadic — COMMENTED** [review]
Hi @esgg - many thanks for your PR. I have left you some comments for consideration.

**Comment by dgarijo** [comment]
thanks @anenadic

File: pages/your_tasks/how_to_make_a_good_readme.md (hunk 8 above)
Comment (dgarijo) [reply pages/your_tasks/how_to_make_a_good_readme.md:71]: I understand this ad the packages installed in the operating system itself. For example, you must have cmake, you  must have a separate visualization software, etc.

File: pages/your_tasks/how_to_make_a_good_readme.md (hunk 2 above)
Comment (dgarijo) [reply pages/your_tasks/how_to_make_a_good_readme.md:24]: I will merge for now, and we can do another review

**Pull request merged**